scripts/get_recs.py

```diff
@@ -0,0 +1,31 @@
+"""
+get_recs.py
+Input: 
+    - sgenres: seed genre
+    - limit: num of songs
+    - country: country code
+    - target_attributes: a dictionary of target audio features
+Output: returns a list of recommended tracks, a list of song names, a list of track IDs
+"""
+import spotipy
+from spotipy.oauth2 import SpotifyOAuth
+from .spotifyoauth import get_token
+from ..common import session
+
+
+def get_recs(sgenres, limit, target_attributes):
+    # get token from session
+    session['token_info'],authorized = get_token()
+    if authorized:
+        sp = spotipy.Spotify(auth=session.get('token_info').get('access_token'))
+    # get recommended tracks 
+    recs = sp.recommendations(seed_genres=sgenres, limit=limit, country='US', **target_attributes)
```

Comment:
Nit: should line 22 & down be indented by 1 more level (inside the `if authorized` branch)? 

Otherwise, if `authorized` is null or false, this line will still execute (and try to use `sp` - which we don't have if the user isn't authorized)